File: c#/Empleados/Comercial.cs
namespace Empleados{
    class Comercial : Empleado{
        private double comision ;

        public Comercial(string nombre,int edad,double salario,double comision):base(nombre,edad,salario){
            this.Salario = this.Salario + comision;
            if(edad > 30 && comision > 200){
                this.Salario = this.Salario + this.PLUS;
            }
            this.comision = comision;
        }

        public override string ToString()
        {
            return base.ToString() + "\nComision: " + this.comision;
        }

    }
}

File: dart/Empleados/Comercial.dart
import "Empleado.dart";

class Comercial extends Empleado{

  // ignore: unused_field
  double? _comision;

  Comercial(nombre,edad,salario,comision):super(nombre,edad,salario){
    this._comision = comision;
    if(edad > 30 && comision > 200){
      this.setSalario = salario + comision + this.PLUS;
    }else{
      this.setSalario = salario + comision;
    }
  }

  @override
  String toString() {
    return super.toString() + "\nComision: " + this._comision.toString();
  }

}

